client/my-sites/media-library/media-folder-dropdown.jsx

```diff
@@ -0,0 +1,102 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { bindAll } from 'lodash';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+/**
+ * Internal dependencies
+ */
+import SelectDropdown from 'components/select-dropdown';
+
+export class MediaFolderDropdown extends Component {
+	static propTypes = {
+		initialSelected: PropTypes.oneOfType( [ PropTypes.string, PropTypes.number ] ),
+		folders: PropTypes.arrayOf(
+			PropTypes.shape( {
+				ID: PropTypes.oneOfType( [ PropTypes.string, PropTypes.number ] ).isRequired,
+				URL: PropTypes.string,
+				children: PropTypes.number.isRequired,
+				date: PropTypes.string,
+				file: PropTypes.string,
+				name: PropTypes.string.isRequired,
+				path: PropTypes.string.isRequired,
+				thumbnails: PropTypes.object,
+				type: PropTypes.string,
+			} )
+		),
+		folder: PropTypes.string,
+		onFolderChange: PropTypes.func,
+		disabled: PropTypes.bool,
+		compact: PropTypes.bool,
+		defaultOption: PropTypes.object,
+	};
+
+	static defaultProps = {
+		initialSelected: '/',
+		folders: [],
+		compact: true,
+	};
+
+	constructor( props ) {
+		super( props );
+
+		bindAll( this, [ 'handleOnSelect' ] );
+	}
+
+	handleOnSelect( option ) {
+		if ( this.props.disabled ) return;
+
+		const folder = option.value;
+
+		this.props.onFolderChange( folder );
+	}
+
+	getDropDownOptions( folderData ) {
+		const separator = null;
+
+		return [
+			{
+				value: '/',
+				label: this.props.translate( 'All Albums' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Albums' )` *ES Score: 19*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).